run/grpc-hello-world-bidi-streaming/app.json

```diff
@@ -0,0 +1,5 @@
+{
+    "options": {
+        "http2": true
```

Comment:
For anyone wondering, this is a configuration file for [Cloud Run button](https://github.com/GoogleCloudPlatform/cloud-run-button#customizing-deployment-parameters)